examples/local_sgd/resnet/train_gpu.sh

```diff
@@ -15,12 +15,12 @@ MODEL_SAVE_PATH="output/"
 
 # training params
 NUM_EPOCHS=90
-BATCH_SIZE=32
-LR=0.001
+BATCH_SIZE=128
+LR=0.1
 LR_STRATEGY=piecewise_decay
 
 # data params
-DATA_PATH="./ImageNet"
+DATA_PATH="/ssd2/shenliang/ImageNet"
```

Comment:
donot use absolute path. And u should write some note to make user aware this data configuration

Reply:
test

Reply:
test2